FreshAir/UI/RZFReleaseNoteCollectionViewCell.m

```diff
@@ -50,28 +69,52 @@ - (instancetype)initWithFrame:(CGRect)frame
     
     if ( self ) {
         self.backgroundColor = [UIColor clearColor];
-        
-        [self setupScreenshotImageView];
-        [self setupInfoContainerView];
     }
     
     return self;
 }
 
 # pragma mark - Setup
 
+- (void)prepareForReuse {
+    [self.releaseNoteTitleLabel removeConstraints:self.releaseNoteTitleLabel.constraints];
```

Comment:
You shouldn't removeConstraints explicitly, `removeFromSuperview` will remove any constraints that it needs to. Generally removing constraints via `.constraints` is also dangerous, because intrinsicContentSize and some other UIKit constraints are in there that shouldn't be managed by hand.